src/flows/components/panel/FlowPanel.tsx

```diff
@@ -144,17 +150,18 @@ const FlowPanel: FC<Props> = ({id, controls, persistentControls, children}) => {
                   />
                 </FeatureFlag>
                 <FeatureFlag name="flow-sidebar">
-                  <SquareButton
+                  <Button
                     icon={IconFont.CogThick}
                     onClick={toggleSidebar}
                     color={
                       id === focused
                         ? ComponentColor.Secondary
                         : ComponentColor.Default
                     }
-                    titleText="Configure"
                     className="flows-config-panel-button"
-                  />
+                    testID="square-button"
+                  ></Button>
```

Comment:
why like this? Why not have the button be a self-closing tsx?

Reply:
Good question lol, ill change that